### PR TITLE
Fix browser tests

### PR DESCRIPTION
**Pull request by r4vi**
This commit makes sure the browser tests pass on latest master as they have been
broken for a while.

Also fixes small bug with the removing `,` logic that always assumes the keys
"value" and "quantity" are in the post_data.

### CODE REVIEW

File: ui_tests/exporter/step_defs/test_standard_application.py [hunk 1]
```diff
@@ -98,7 +98,7 @@ def i_add_a_non_incorporated_good_to_the_application(driver, context):  # noqa
     StandardApplicationGoodsPage(driver).click_add_preexisting_good_button()
 
     # Click the "Add to application" link on the first good
-    driver.find_elements_by_css_selector(".govuk-table__row .govuk-link")[0].click()
+    driver.find_elements_by_id("add-to-application")[0].click()
```
Comment (r4vi) [inline ui_tests/exporter/step_defs/test_standard_application.py:101]: this handles both the FIREARMS_ENABLED FF on or off

File: unit_tests/exporter/goods/test_forms.py [hunk 2]
```diff
@@ -46,3 +46,25 @@ def test_serialize_good_on_app_data(value, serialized):
...
+        ("2,300.00", "2300.00"),
+        ("2,3,,,,,00.00", "2300.00"),
+        ("23,444200", "23444200"),
+        ("foo", "foo"),  # this will be caught by serializer on the api and return an error
+        ("84.34.111", "84.34.111"),  # this too
+    ],
+)
+def test_serialize_good_on_app_data_no_value_key(value, serialized):
+
+    data = {
```
Comment (r4vi) [inline unit_tests/exporter/goods/test_forms.py:62]: if `post_data` doesn't have `value` or `quantity` this function used to blow up with a KeyError

File: exporter/applications/services.py [hunk 3]
```diff
@@ -119,7 +119,7 @@ def serialize_good_on_app_data(json):
         post_data = json
     if "good_id" not in post_data:
         post_data["good_id"] = json["good_id"]
-    for key in ["value", "quantity"]:
+    for key in {"value", "quantity"} & set(post_data.keys()):
```
Comment (r4vi) [inline exporter/applications/services.py:122]: `&` is set.intersection operator

**Review by kerin — APPROVED** [review]
👍

**Review by saruniitr — APPROVED** [review]
looks good to me

**Pull request merged**